internal/state/graph/httproute_test.go

```diff
@@ -537,6 +537,7 @@ func TestBindRouteToListeners(t *testing.T) {
 			Source: v1beta1.Listener{
 				Name:     v1beta1.SectionName(name),
 				Hostname: (*v1beta1.Hostname)(helpers.GetStringPointer("foo.example.com")),
+				Port:     80,
```

Comment:
why are we setting this? we don't support this field of the parentRef

Reply:
This is the listener port, not the parentRef port.

Reply:
Ah, I see what you mean. I'll remove

Reply:
> This is the listener port, not the parentRef port.

oh, I misread it :( 
> Ah, I see what you mean. I'll remove

I wish I knew :)

Reply:
lol. The listener port isn't needed for this test because we don't check the listener port when binding a route to a listener.